nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config

```diff
@@ -24,7 +24,7 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = "./iGenomes"
+  igenomes_base = 's3://ngi-igenomes/igenomes/'
```

Comment:
Woah, this has been in the template all along??

Reply:
Yep.

Reply:
😱

Reply:
`base.config` must have priority! 🤣

Reply:
Yeah, I think it's loaded after this block.. Bit scary!